src/python/pants/goal/run_tracker.py

```diff
@@ -103,6 +103,14 @@ def __init__(self, args: tuple[str, ...], options: Options):
     def goals(self) -> list[str]:
         return self._all_options.goals if self._all_options else []
 
+    @property
+    def active_standard_backends(self) -> list[str]:
+        return [
+            backend
+            for backend in self._all_options.for_global_scope().backend_packages
+            if backend.starts_with("pants.backend.")
```

Comment:
Should this be compared instead against an explicit list of backends to ensure that users that have custom backends on the PYTHONPATH and chose to put it in `pants.backend.*` do not have those backends reported?

Reply:
I think this is fine as-is. If you use our namespace it's fair game.